sdk/instance/sdk.go

```diff
@@ -35,26 +38,17 @@ func (s *SDK) Get(hash hash.Hash) (*instance.Instance, error) {
 }
 
 // List returns all instances.
-func (s *SDK) List(f *Filter) ([]*instance.Instance, error) {
-	var instances []*instance.Instance
-	if err := s.client.Query("custom/"+backendName+"/list", nil, &instances); err != nil {
+func (s *SDK) List(f *api.ListInstanceRequest_Filter) ([]*instance.Instance, error) {
+	fb, err := json.Marshal(f)
```

Comment:
could you move the marshal inside the client.Query. it already unmarshal the received data.